xbmc/video/guilib/VideoSelectActionProcessor.cpp

```diff
@@ -70,7 +71,10 @@ bool CVideoSelectActionProcessorBase::Process(SelectAction selectAction)
       return OnResumeSelected();
 
     case SELECT_ACTION_PLAY:
+    {
+      ChooseVideoVersion();
```

Comment:
There is also https://github.com/xbmc/xbmc/blob/master/xbmc/video/guilib/VideoPlayActionProcessor.h, wich might need to be adjusted as well.

Reply:
Will add that.